### PR TITLE
Add columns to the Silo endpoint

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
The users should be able to see the columns of the table in the silo endpoint. It'll be easier for the name to handle the data afterward.

## Approach
I only added the columns field to the serializer.

### Furter Info
Related ticket: #440

### CODE REVIEW

File: silo/api.py [hunk 1]
```diff
@@ -274,24 +274,25 @@ class SiloViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = SiloSerializer
     lookup_field = 'id'
     # this permission sets seems to break the default permissions set by the restframework
-    # permission_classes = (IsOwnerOrReadOnly,)
+    # permission_classes = (IsOwnerOrReazzdOnly,)
```
Comment (leonholub) [inline silo/api.py]: Why is this in here if it is commented out. And why did you change it to `IsOwnerOrReazzdOnly` ??

Comment (jefmoura) [reply silo/api.py]: I didn't see this! hahaha
